.travis.yml

```diff
@@ -2,12 +2,13 @@ sudo: required
 language: csharp
 mono:
   - 5.8.0
-solution: QuantConnect.Lean.sln
+solution: QuantConnect.Lean.VS17.sln
 install:
-  - nuget restore QuantConnect.Lean.sln
   - nuget install NUnit.Runners -Version 2.6.4 -OutputDirectory testrunner
 script:
-  - xbuild /p:Configuration=Release /p:VbcToolExe=vbnc.exe QuantConnect.Lean.sln
+  - msbuild /p:Version="2.4.0.1" /t:restore,build /p:Configuration=Release QuantConnect.Lean.VS17.sln
   - mono ./testrunner/NUnit.Runners.2.6.4/tools/nunit-console.exe ./Tests/bin/Release/QuantConnect.Tests.dll --exclude:TravisExclude --labels
 after_success:
-  - coveralls
\ No newline at end of file
+  - coveralls  
+  - curl -O https://dist.nuget.org/win-x86-commandline/latest/nuget.exe
+  - find . -name *.nupkg -not -path "*/packages/*" -not -path "*/testrunner/*"  -exec mono ./nuget.exe push {} $mygetApiKey  -Source https://www.myget.org/F/leantest/api/v2/package \;
```

Comment:
This is really cool and we're going to borrow that command line :D But we've got a planned nuget system @Jay-Jay-D is building now that ties into our "build numbers". In the cloud you'll see build numbers in the console -- we're hoping to tie them all together. Would you mind removing this bit?

Reply:
![image](https://user-images.githubusercontent.com/2243225/38588517-5a6c2e90-3cf4-11e8-9d36-70afbb12abe0.png) the 3471 is the build number on the end.

Reply:
Sure, I can remove it, I take that the nuget generation should also be disabled? As currently there is a flag in the csproj files that generates the nuget.

The planned nuget system is a lot different? I couldn't find that line in the build log, though if you want the build number of travis behind the version in the nuget then the above line of /p:Version="2.4.0.1" could be changed to something like /p:Version="2.4.0.1.$((TRAVIS_JOB_ID - 1))", in addition a condition could be added that it only pushes for the master branch.

Reply:
Thanks yes please remove that flag too. We're ready to go with the automated publishing just need your assistance transferring the unofficial packages over.